### ISSUE TITLE
Can't Produce a new Package Version where Dependency has Email Template and Folder

### ISSUE THREAD

**Issue by CXCAlfo** [forcedotcom/cli#1043]
### Summary

As of this morning I am receiving an error when creating an unlocked package version that is dependent on a single other package.  The error involves an email template for community but we haven't added any new email templates recently, 4 months ago was the last time and we have packaged constantly since then. 

### Steps To Reproduce:

1. Create a new scratch with our definition file - nothing really special and is the same definition used since January.
[enterprise-scratch-def.txt](https://github.com/forcedotcom/cli/files/6643335/enterprise-scratch-def.txt)
1. Create a Package Version for an existing package - this is the foundation package and contains objects, emails etc.
![EmailFolder](https://user-images.githubusercontent.com/59152325/121796635-f8047a80-cc5d-11eb-81f5-4a27f3f1672e.PNG)  **This package creates successfully with no errors.**
1. Create the 2nd Package Version that is dependent on the first package (no email folders in 2nd package)

### Expected result

Creates a successful package version as it has done since January

### Actual result

An error occurred while trying to install a package dependency, ID 04t..... Experience Cloud: User Lockout Email: Experience Cloud: User Lockout Email file is located in both the package you tried to install, and in either the unfiled-public folder or a user folder in this org. 
To install this package, either delete Experience Cloud: User Lockout Email from this org, or move it to a different folder.

### Additional information
SFDX CLI Version = sfdx-cli/7.105.0 win32-x64 node-v14.17.0

@oclif/plugin-autocomplete 0.3.0 (core)
@oclif/plugin-commands 1.3.0 (core)
@oclif/plugin-help 3.2.2 (core)
@oclif/plugin-not-found 1.2.4 (core)
@oclif/plugin-plugins 1.10.0 (core)
@oclif/plugin-update 1.4.0-3 (core)
@oclif/plugin-warn-if-update-available 1.7.0 (core)
@oclif/plugin-which 1.0.3 (core)
@salesforce/sfdx-trust 3.6.0 (core)
alias 1.1.10 (core)
auth 1.6.1 (core)
config 1.2.8 (core)
generator 1.1.7 (core)
salesforcedx 52.0.0 (latest-rc)
├─ data 0.4.11
├─ apex 0.2.2
├─ custom-metadata 1.0.12
├─ limits 1.2.1
├─ user 1.3.0
├─ schema 1.0.7
├─ org 1.6.6
├─ @salesforce/sfdx-plugin-lwc-test 0.1.7
├─ templates 51.5.0
└─ salesforce-alm 52.0.0
sfdmu 4.4.2
sfdx-cli 7.105.0 (core)
sfdx-git-delta 4.5.0
telemetry 1.2.1 (core)

Windows 10

**Comment by CXCAlfo** [comment]
As a quick update I have found what causes the problem but not sure "why" i.e. where the problem happens.

I created a new scratch and did nothing to it, the scratch has 7 templates in the unfiled folder, I don't know if this is new or always been there.
![Emails](https://user-images.githubusercontent.com/59152325/121802706-162fa200-cc81-11eb-9259-26d73d9f725a.PNG)

I went back to an old branch that I packaged on Friday before V52 was implemented and it failed for the same reason.

Once I remove the 6 highlighted emails from my folder (they have been there for 4 months) I get it all to package.
